modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -580,6 +583,18 @@ export class Utils implements BaseUtils {
       sk: keys.secretKey,
     };
   }
+
+  getTxID(txBase64: string): string {
```

Comment:
In this case I think getTxId is not the best name because we will only use this util for doubleSpend @pancham88 may be here we need another proper name for this method. Also please add js doc.